src/mscorlib/src/System/IO/Stream.cs

```diff
@@ -900,6 +900,13 @@ private sealed class NullStream : Stream
                 get { return 0; }
                 set {}
             }
+            
+            public override Task CopyToAsync(Stream destination, int bufferSize, CancellationToken cancellationToken)
+            {
+                if (cancellationToken.IsCancellationRequested)
```

Comment:
@jkotas Well maybe, but `Stream.Null` already overrides `ReadAsync`/`WriteAsync` for perf, so why not `CopyToAsync` as well?